sirepo/template/flash.py

```diff
@@ -480,7 +481,7 @@ def _parameters(f):
     res = {}
     for name in ('integer scalars', 'integer runtime parameters', 'real scalars', 'real runtime parameters'):
         for v in f[name]:
-            res[v[0].strip()] = v[1]
+            res[pkcompat.from_bytes(v[0].strip())] = v[1]
```

Comment:
Other codes don't need to do this with h5py. Why is this happening here?

Reply:
The honest truth is I don't really know. Below is what I do know.

The value `v` is a compound datatype (`ascii str`, `int32`). We have to parse the ascii str ourselves. The ascii str then becomes the key for the dictionary. In other cases that I tested (ex warpvnd) it seems we only use path names as the keys for the dictionary we never use data memebers so we never have to do any parsing ourselves. Should I do more digging or is that good?